src/app/page.tsx

```diff
@@ -29,3 +26,30 @@ export const metadata: Metadata = {
       '광주소프트웨어마이스터고등학교 학생들의 취업 정보를 확인 할 수 있어요.',
   },
 };
+
+const getWorkerList = async (): Promise<WorkerType[]> => {
+  const cookieStore = cookies();
+
+  const accessToken = cookieStore.get('accessToken')?.value;
+
+  try {
+    const response = await fetch(
+      new URL('/api/worker/list', process.env.API_BASE_URL),
+      {
+        headers: {
```

Comment:
```suggestion
      {
        method: 'GET',
        headers: {
```

요청의 method를 명시적으로 적어주는 것이 좋을 것 같습니다!

Reply:
좋은 제안 감사합니다

[9abe702](https://github.com/GSM-Networking/GSM-Networking-client/pull/12/commits/9abe702430ea1b5a8e8efe1ede6a65b10642f744)

수정했습니다